### PR TITLE
Login form

### PR DESCRIPTION
**Pull request by Ragsomar**

### CODE REVIEW

File: front/src/Components/LoginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        this.setState({
+            [e.target.name]: e.target.value
+        })
+    }
+
+    submitForm(e) {
+        e.preventDefault()
+    }
+
+    postForm() {
```
Comment (rocambille) [inline front/src/Components/LoginForm.jsx]: why not merge submitForm and postForm?

**Review by rocambille — CHANGES_REQUESTED** [review]

File: front/src/Components/LoginForm.jsx (hunk 1 above)
Comment (Ragsomar) [reply front/src/Components/LoginForm.jsx]: Indeed, just did it

File: front/src/Components/LoginForm.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                            name='password'
+                            onChange={this.onChange}
+                            value={this.state.password}
+                            required
+                        />
+                    </div>
+                    <input
+                        type='submit'
+                        value='Envoyer'
+                        onClick={this.submitForm}
```
Comment (rocambille) [inline front/src/Components/LoginForm.jsx]: redundant with `<form onSubmit={this.submitForm} ...`

**Review by rocambille — CHANGES_REQUESTED** [review]

File: front/src/Components/LoginForm.jsx (hunk 2 above)
Comment (Ragsomar) [reply front/src/Components/LoginForm.jsx]: Done, didn't know it could work this way

**Pull request merged**